docs/contribute/_snippets/changelog-fields.md

```diff
@@ -0,0 +1,103 @@
+
+```yaml
+##### Required fields #####
+title:
+
+# A required string that is a short, user-facing headline
+# Max 80 characters
+
+type: 
+
+# A required string that contains the type of change.
+# It can be one of:
+# - feature: new functionality
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - bug-fix: fixes a problem in a previous version
+# - known-issue: problems that we are aware of in a given version
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - docs: major documentation updates or reorgs
+# - regression: functionality that no longer works or behaves incorrectly
+# - security: an advisory about potential security vulnerabilities
+# - other: does not fit into any of the other categories
+
+products:
+
+# A required array of objects that denote the affected products and their target release.
```

Comment:
I'd add a full example here as well.

Reply:
I'm not going to add examples in that snippet, but we'll add more in https://docs-v3-preview.elastic.dev/elastic/docs-builder/pull/2320/contribute/changelog#examples in subsequent PRs.